ADD changelog (#9)
plentymarkets/plugin-google-analytics: meta/documents/changelog_en.md

# Release Notes for GoogleAnalytics

## v1.1.0 (2019-12-03)
### Changes

> - Cookies used by Google Analytics are registered on the consent interface for Ceres webstores.

Reviewer: ```suggestion
- Cookies used by Google Analytics are registered with the consent interface for Ceres online stores.
```
